### PR TITLE
Added tests to verify a new label to communicate latest introspect version

### PR DESCRIPTION
**Pull request by hzhao-github**
use case:

1. When a domain resource has spec.introspectVersion configured, all WebLogic server pods will have a label "weblogic.introspectVersion" set to the value of spec.introspectVersion.
2. When a domain resource has spec.introspectVersion configured, after a cluster is scaled up, new server pods have the label "weblogic.introspectVersion" set as well.
3. When the spec.introspectVersion is changed, all running server pods' weblogic.introspectVersion label is updated to the new value.
4. When a domain/cluster is rolling restarted without changing the spec.introspectVersion, all server pods' weblogic.introspectVersion label stay unchanged after the pods are restarted.

Passed Jenkins jobs:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/3095   (sequential)
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/3109/testReport/  ( parallel)

failed test oracle.weblogic.kubernetes.ItMonitoringExporter.testEndToEndViaChart is a knowing issue

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java [hunk 1]
```diff
@@ -1008,6 +1062,46 @@ private static void verifyMemberHealth(String adminServerPodName, List<String> m
         });
   }
 
+  private void verifyWebLogicVersioninPod(int replicaCount) {
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:1065]: Change the method name to verifyIntrospectVersionLabelInPod()  to represent the action.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java [hunk 2]
```diff
@@ -1008,6 +1062,46 @@ private static void verifyMemberHealth(String adminServerPodName, List<String> m
...
+
+    // verify managed server pods
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Verify weblogic.introspectVersion in managed server pod {0}",
+          managedServerPodNamePrefix + i);
+      verifyWebLogicVersionValue(managedServerPodNamePrefix + i, introspectVersion);
+    }
+  }
+
+  private void verifyWebLogicVersionValue(String podName, String introspectVersion) {
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:1086]: Change the method name to verifyIntrospectVersionLabelValue() to represent the action

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java (hunk 1 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:1065]: Do we set the IntrospectVersion explicitly on domain resource using  spec.introspectVersion syntax  while creating the domain ?

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java]: changed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java (hunk 2 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java]: changed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java (hunk 1 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java]: yes

**Review by anpanigr — APPROVED** [review]
LGTM

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java [hunk 3]
```diff
@@ -263,6 +263,22 @@ public static boolean patchDomainResourceWithNewIntrospectVersion(
...
+   * Get current introspectVersion for a given domain.
+   *
+   * @param domainUid domain id
+   * @param namespace namespace in which the domain resource exists
+   * @return String containing current introspectVersion
+   * @throws ApiException when getting domain resource fails
+   */
+  public static String getCurrentIntrospectVersion(String domainUid, String namespace) throws ApiException {
+    LoggingFacade logger = getLogger();
+    oracle.weblogic.domain.Domain domain = Domain.getDomainCustomResource(domainUid, namespace);
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java:276]: no implementation in TestActions.java. create a method in Domain class and call that from here.

**Review by vanajamukkara — CHANGES_REQUESTED** [review]
LG, left one comment to address.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java (hunk 3 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java]: changed

**Review by vanajamukkara — APPROVED** [review]

**Pull request merged**